x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/use_response_actions_log_table.tsx

```diff
@@ -69,7 +69,7 @@ const StyledDescriptionList = euiStyled(EuiDescriptionList).attrs({
 
 // output section styles
 const topSpacingCss = css`
-  ${(props) => `${props.theme.eui.euiCodeBlockPaddingModifiers.paddingMedium} 0`}
+  ${(props) => `${props.theme.eui.euiSize} 0`}
```

Comment:
Not related to this line of change, but for some reason, the `font-weight` in the description list (`customDescriptionListCss` lines 48-61) is indeed heavier (semiBold). Perhaps `euiFontWeightSemiBold` was being overwritten by something else in previous versions and with this update, it isn't.


Basically, the weight should be normal and the list should be seen like so,

![Screenshot 2022-11-18 at 10 24 34](https://user-images.githubusercontent.com/1849116/202671337-ce049d0d-6342-4bd1-984b-88ba238ba7b6.png)

Yet, in this update branch, it shows up like this:

![Screenshot 2022-11-18 at 09 50 12](https://user-images.githubusercontent.com/1849116/202671441-59b92f33-5e82-44c0-b31a-bc18c44ee662.png)


If you replace lines 48-61 above, with the following it should fix that.

```
const customDescriptionListCss = css`
  &.euiDescriptionList {
    > .euiDescriptionList__title {
      color: ${(props) => props.theme.eui.euiColorDarkShade};
      font-size: ${(props) => props.theme.eui.euiFontSizeXS};
    }

    > .euiDescriptionList__title,
    .euiDescriptionList__description {
      font-weight: ${(props) => props.theme.eui.euiFontWeightRegular};
      margin-top: ${(props) => props.theme.eui.euiSizeS};
    }
  }
`;
```

Reply:
Addressed in https://github.com/elastic/kibana/pull/144845/commits/d5f93d327e153105f57dabe1d981ee8ce3404527 - thanks for the find and the quick fix!

Yes, in our Emotion conversions, the majority of our CSS should now be much flatter and have less nesting/specificity, so CSS that was previously _not_ being overridden may now start overriding our CSS. However, the upside is it should be slightly less painful to override EUI CSS in the future.

Reply:
@ashokaditya It's worth noting that `EuiDescriptionList` was not converted to Emotion in this EUI upgrade, meaning that this bug has been around for a while. Looking back at our changelogs, it looks like the Emotion conversion of `EuiDescriptionList` made it into 8.5, so you will need to backport to both 8.5 and 8.6 if the visual bug bothers you enough to want to fix it in past releases.